corrected sentence (#56595)
vercel/next.js: docs/02-app/01-building-your-application/02-data-fetching/index.mdx

---
title: Data Fetching

> description: Learn how to fetch, cache, revalidate, and mutate data in React and Next.js.

Reviewer: ```suggestion
description: Learn how to fetch, cache, revalidate, and mutate data with Next.js.
```
